docs/sphinx/source/changelog/pending.rst

```diff
@@ -0,0 +1,38 @@
+************************
+Pending
+************************
+
+API Changes
+-----------
+
+
+Enhancements
+------------
+* Add new function :py:func:`~rdtools.soiling.monthly_soiling_rates` (:pull:`193`).
+* Add new function :py:func:`~rdtools.annual_soilng_ratios` (:pull:`193`).
+
+
+Bug fixes
+---------
+
+
+Testing
+-------
+
+
+Documentation
+-------------
+
+Requirements
+------------
+
+
+Example Updates
+---------------
+* :py:func:`~rdtools.soiling.monthly_soiling_rates` added to degradation_and_soiling_example_pvdaq_4.ipynb
```

Comment:
I think `annual_soiling_ratios` was added as well, no?

Reply:
#196 added some new notebooks -- we should think about how we want to keep things in sync between them.  It's a little inefficient to have code duplication between notebooks, but maybe that's just what we need to do.

Reply:
Agreed, but I'm not sure we have a great option for avoiding it at this point. Let's make a point to look back and run all the notebooks whenever we merge to master, that will at least give us a chance to get things in sync manually.